Add  volume converter package

kmilo9999 opened the pull request.

kmilo9999 committed:
add volume-data-converter

kmilo9999 committed:
going up on volume-data-converter 2.1.1

tdivoll reviewed (approved):
Looks pretty straight forward to me! So the package will live in it's own repo?

mirestrepo reviewed (changes requested):
This looks good, but I would add the related notebook in this PR. There is no need for these dependencies unless there is a notebook that is using them

kmilo9999 committed:
added volume viewer jupiter notebook

kmilo9999 committed:
updating requirements.txt

kmilo9999 committed:
merge with main

kmilo9999 committed:
Update firebase-hosting-merge.yml

changed python version to 3.7.9 (support google colab)

kmilo9999 committed:
Update firebase-hosting-pull-request.yml

updated python version to 3.7.9 (support google colab)

kmilo9999 committed:
added open google colab button

kmilo9999 committed:
Merge branch 'feature-volume-converter' of https://github.com/ridatadiscoverycenter/riddc-jbook into feature-volume-converter

kmilo9999 committed:
added volume viewer section in the _toc.yml project

kmilo9999 committed:
added images for the volume viewer notebook

kmilo9999 committed:
added new image

kmilo9999 committed:
updated image

kmilo9999 committed:
updated image

kmilo9999 committed:
added rescaled image

kmilo9999 committed:
renamed files and deleted image

kmilo9999 committed:
fixed url typo

kmilo9999 commented:
@mirestrepo   I finally added the 3d volume viewer notebook working on google colab.

kmilo9999 commented:
@eldu can you help me with the English text in the Volume_viewer_converter.ipynb notebook?  thanks

eldu commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+    "\n",
+    "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/volume-viewer-2.png\" >\n",
+    "\n",
+    "You can download the latest version of the volume viewer application on [this link](https://github.com/brown-ccv/VR-Volumeviewer/releases)\n",
```
```suggestion
    "You can download the latest version of the volume viewer application from [our latest releases](https://github.com/brown-ccv/VR-Volumeviewer/releases).\n",
```

kmilo9999 replied:
Thank you Ellen, I added your suggestions.

eldu commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+    "\n",
+    "You can download the latest version of the volume viewer application on [this link](https://github.com/brown-ccv/VR-Volumeviewer/releases)\n",
+    "\n",
+    "We have created the following example on how to export osom datasets and visualize them in 3D using the volume viewer application.\n",
```
```suggestion
    "The following example goes through how to export osom datasets and visualize them in 3D using the volume viewer application.\n",
```

eldu commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+    "You can download the latest version of the volume viewer application on [this link](https://github.com/brown-ccv/VR-Volumeviewer/releases)\n",
+    "\n",
+    "We have created the following example on how to export osom datasets and visualize them in 3D using the volume viewer application.\n",
+    "These scripts create large amount of data, and we recommend to use your google drive to save them. All the instructions here explains how to proceed.\n",
```
```suggestion
    "**NOTE: These scripts create large amount of data. We recommend to use your google drive to save them.**\n",
```

eldu commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+    "id": "yBxXaGPG9Cpf"
+   },
+   "source": [
+    "Optional: The previous code works most of the times. If you experience any issues and want to verify the book can see your google drive file system, execute the following code. It should display a list of files located in your google drive."
```
```suggestion
    "Optional: If you experience any issues and want to verify the book can see your google drive file system, execute the following code. It should display a list of files located in your google drive."
```

eldu commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+    "id": "HDwcgzexF_6g"
+   },
+   "source": [
+    "Now, we need to download the nc grid file to map our global data points to the application coordinates. Execue the following command.\n"
```
```suggestion
    "Now, we need to download the nc grid file to map our global data points to the application coordinates. Execute the following command.\n"
```

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>Should we save these in google drive? Seems useful to have it for future instances? Also should we make a folder to store all the work from this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

kmilo9999 replied:
<p>The files in the example (.json and nc files) are saved in the session <code>./content</code>  folder. It gets deleted after the session expiries. Only the volume viewer raw files are saved into google drive</p><p></p>

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>Add headings. i.e</p><p> </p><h1>Install volume converted package</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

kmilo9999 replied:
<p>Done</p>

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>Should this be saved to Drive as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

kmilo9999 replied:
<p>Same as above. This example file is not saved into google drive, but the temporary session folder</p>

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>I don't think that osom_grid_file is inside Gdrive. Maybe you are missing a</p><p> </p><pre class="ql-syntax" spellcheck="false">cd ./gdrive/MyDrive
mkdir osom3d

</pre><p>Then save everything in there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

kmilo9999 replied:
<p>You are very right, I fixed the path.</p>

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>It will be good to show the output here? Not sure why it is omitted</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

kmilo9999 replied:
<p>Good point. I left it unintentionally. I added the output.</p>

mirestrepo reviewed (changes requested):
This is looking great!!!
I submitted some suggestions to the notebook using reviewNB. Let's see how those show up. There is an image file called screenshot, rename t more specific name. The notebook filename has an upper case V, make all lowercase and use `git mv` to rename to avoid future problems in case insensitive OS

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,355 @@
+{
```
<p>Separate into sections</p><p> </p><h1>1- Download Volume Viewer</h1><p>text</p><p> </p><h1>2- Mount Google Drive</h1><p> </p><p>Also the link to download volume viewer is not self-explanatory. There is a zip in there. What should pleople do? Get executables, instructions to Oscar?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

kmilo9999 replied:
<p>done</p>

kmilo9999 committed:
added sections and explanations on volume viewer install

eldu reviewed (commented):
Well it turns out i never submitted my review. here it is !

kmilo9999 committed:
fix markdown spaces

kmilo9999 committed:
Apply suggestions from code review

Ellen's notes

Co-authored-by: Ellen Duong <shi.ellenduong@gmail.com>

kmilo9999 committed:
fixed header on section 1

kmilo9999 committed:
Merge branch 'feature-volume-converter' of https://github.com/ridatadiscoverycenter/riddc-jbook into feature-volume-converter

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,377 @@
+{
```
<p>Should OSOM be all CAPs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

kmilo9999 replied:
I changed the syntax to OSOM.

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,377 @@
+{
```
<p>There is a space missing in the heading after ## so that it shows as a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

kmilo9999 replied:
done

kmilo9999 committed:
added commands to save nc and json files to google drive

kmilo9999 committed:
removed unnecessary output from executed commands

kmilo9999 committed:
added support for python >3.7 , <3.11

kmilo9999 committed:
Update firebase-hosting-pull-request.yml

moving up on python form 3.7 to 3.9

kmilo9999 committed:
fixed markdown syntax.

kmilo9999 committed:
Merge branch 'feature-volume-converter' of https://github.com/ridatadiscoverycenter/riddc-jbook into feature-volume-converter

kmilo9999 committed:
Update firebase-hosting-merge.yml

moving up on python form 3.7 to 3.9

kmilo9999 committed:
Merge branch 'feature-volume-converter' of https://github.com/ridatadiscoverycenter/riddc-jbook into feature-volume-converter

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/volume-viewer-2.png\" >\n",
+    "\n",
+    "The following example goes through how to export Osom datasets and visualize them in 3D using the volume viewer application.\n",
```
```suggestion
    "The following example goes through how to export OSOM datasets and visualize them in 3D using the volume viewer application.\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/volume-viewer-2.png\" >\n",
+    "\n",
+    "The following example goes through how to export Osom datasets and visualize them in 3D using the volume viewer application.\n",
+    "**NOTE: These scripts create large amount of data. We recommend to use your google drive to save them.**\n",
```
```suggestion
    "**NOTE: These scripts create a large amount of data. We recommend using your Google Drive to save them.**\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "You can use the volume viewer either from you computer or using Brown's Oscar HPC.\n",
+    "\n",
+    "## 1 Get the Volume Viewer Application\n",
+    "\n",
```
```suggestion
    "\n",
    "\n",
```

tdivoll replied:
I think this will work, just adding another space under the heading to make the section stand out more

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "### b. Using Oscar HPC\n",
+    "\n",
+    "Loggin in to Oscar using [ood](https://ood.ccv.brown.edu/) and open the desktop application with at least 1 GPU.\n",
```
```suggestion
    "Log in to Oscar using [OOD](https://ood.ccv.brown.edu/) and open the desktop application with at least 1 GPU.\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/ood-desktop-session.png\" >\n",
+    "\n",
+    "Go to terminal and type the commands:\n",
```
```suggestion
    "Go to the terminal inside the virtual desktop and type the following commands:\n",
     "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/ood_terminal.png\" >\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "yh5jzMGT9Dnh"
+   },
+   "source": [
+    "***We recommend working with this notebook on Google colab***\n",
```
```suggestion
    "***We recommend working with this notebook on Google Colab***\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "## 2. Mount Google Drive\n",
+    "\n",
+    "Execute the command below to connect to our google drive\n",
```
```suggestion
    "Execute the rest of the commands below directly in this notebook to connect to your Google Drive\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "Execute the command below to connect to our google drive\n",
+    "\n",
+    "**NOTE: The following command will open a pop-up window requesting access to your google drive file system**\n",
```
```suggestion
    "**NOTE: The following command will open a pop-up window requesting access to your Google Drive file system**\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "yBxXaGPG9Cpf"
+   },
+   "source": [
+    "Optional: If you experience any issues and want to verify the book can see your google drive file system, execute the following code. It should display a list of files located in your google drive."
```
```suggestion
    "Optional: If you experience any issues and want to verify that the notebook can access your Google Drive file system, execute the following code. It should display a list of files located in your Google Drive."
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "2_Bn8otEzO0L"
+   },
+   "source": [
+    "Create a folder in google drive to store the configuration, osom-data and results files."
```
```suggestion
    "Create a folder in Google Drive to store the configuration, osom-data, and results files."
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+   "source": [
+    "## 4. Download OSOM data from ERDDAP\n",
+    "\n",
+    "First we need to acquire data from the erddap server. In order to configure a connection and set up constrains for your search, create a json containing the connection url, protocol type and dataset id. You can also specify a time range and coordinates. The following is an example on how to create a json file with the mentioned properties. \n",
```
```suggestion
    "First we need to acquire data from the ERDDAP server. In order to configure a connection and set up constrains for your search, create a json containing the connection url, protocol type and dataset id. You can also specify a time range and coordinates. The following is an example on how to create a json file with the mentioned properties. \n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "gsx2Kmhuwv-F"
+   },
+   "source": [
+    "After running the previous script, if we list the files in our google drive folder, we can see the recently created `'erddap_connection.json'` file.\n"
```
```suggestion
    "After running the previous script, if we list the files in our Google Drive folder, we can see the recently created `'erddap_connection.json'` file.\n"
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "xaVjrrSw1xte"
+   },
+   "source": [
+    "Using the `volume_data_converter` library, we will pass `erddap_connection.json` to the `my_erddap_query` function to start up the to download process and retrieve the data from erddap."
```
```suggestion
    "Using the `volume_data_converter` library, we will pass `erddap_connection.json` to the `my_erddap_query` function to start the download process and retrieve the data from ERDDAP."
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "8AGEdXSG0Tun"
+   },
+   "source": [
+    "If we list the files in our google drive folder, we can see the ouput file `model_data_57db_4a85_81d9.nc` which is the name of the dataset id pointent in the `erddap_connection.json`.\n"
```
```suggestion
    "If we list the files in our Google Drive folder, we can see the ouput file `model_data_57db_4a85_81d9.nc` which is the name of the dataset id specified in the `erddap_connection.json`.\n"
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "wisVJhej1bv-"
+   },
+   "source": [
+    "## 5. Convert NC-OSOM data file to Volume viewer file format\n",
```
```suggestion
    "## 5. Convert NC-OSOM data file to Volume Viewer file format\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+   "source": [
+    "## 5. Convert NC-OSOM data file to Volume viewer file format\n",
+    "\n",
+    "Now we will create another json file containing the configuration to convert the nc file to raw data that the volume viewer app will read. The following code is an example of the type of configuration file the `volume_data_converter`expects to work with:\n",
```
```suggestion
    "Now we will create another json file containing the configuration to convert the nc file to raw data that the volume viewer app will read. The following code is an example of the type of configuration file the `volume_data_converter` expects to work with:\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "- `osom_grid_file`: Path to the nc grid file (the first downloaded file)\n",
+    "- `osom_data_file`: Path to the nc osom data file (`model_data_57db_4a85_81d9.nc`)\n",
+    "- `ouput_folder`: Path where the resulting package will be saved (a place in your google drive file system)\n",
```
```suggestion
    "- `ouput_folder`: Path where the resulting package will be saved (a place in your Google Drive file system)\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "Now we will create another json file containing the configuration to convert the nc file to raw data that the volume viewer app will read. The following code is an example of the type of configuration file the `volume_data_converter`expects to work with:\n",
+    "\n",
+    "- `osom_grid_file`: Path to the nc grid file (the first downloaded file)\n",
+    "- `osom_data_file`: Path to the nc osom data file (`model_data_57db_4a85_81d9.nc`)\n",
```
```suggestion
    "- `osom_data_file`: Path to the OSOM nc data file (`model_data_57db_4a85_81d9.nc`)\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "- `osom_grid_file`: Path to the nc grid file (the first downloaded file)\n",
+    "- `osom_data_file`: Path to the nc osom data file (`model_data_57db_4a85_81d9.nc`)\n",
+    "- `ouput_folder`: Path where the resulting package will be saved (a place in your google drive file system)\n",
+    "- `data_descriptor`: The osom variable to export to the voluem viewer (temp,salt)\n",
```
```suggestion
    "- `data_descriptor`: The OSOM variable to export to the volume viewer (e.g., temp, salt)\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "- `ouput_folder`: Path where the resulting package will be saved (a place in your google drive file system)\n",
+    "- `data_descriptor`: The osom variable to export to the voluem viewer (temp,salt)\n",
+    "- `time_frames`: If the dataset has multiple time frames, it will export all of them (time sequence). Otherwise, specify the required frame. (i.e: Null, [0,1],[5,6,7])\n",
+    "- `layer`: Currently osom supported files have 3 different types of layes `all`(default), `surface`, `bottom`.\n",
```
```suggestion
    "- `layer`: Currently, OSOM-supported files have 3 different types of layers: `all`(default), `surface`, `bottom`.\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "gKY-6EkeAWwP"
+   },
+   "source": [
+    "Take in count the defined ouput file path points to `./gdrive/myDrive/osom`. That is a location inside **your** google drive. From there you can downloaded to the environment you are running the volume viewer application"
```
```suggestion
    "Take into account that the defined output file path points to `./gdrive/myDrive/osom`. That is a location inside **your** Google Drive. From there you can download to the environment in which you are running the volume viewer application."
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "id": "y18TazFKBiDx"
+   },
+   "source": [
+    "## 6. Load data into the volume viewer\n",
```
```suggestion
    "## 6. Load data into the Volume Viewer\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "## 6. Load data into the volume viewer\n",
+    "\n",
+    "\n",
+    "- Go to your google drive web app. In the left side panel you will find the result output folder (see image below). Do right click on the folder and select **Download**.\n",
```
```suggestion
    "- Go to your Google Drive web app. In the left side panel you will find the result output folder (see image below). Right-click on the folder and select **Download**.\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "![google drive folder](https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/Screen%20Shot%20google%20drive%20file%20system.png)\n",
+    "\n",
+    "\n",
+    "- Unzip the file and open the Volume viewer application.\n",
```
```suggestion
    "- Unzip the file on your local computer and navigate to the Volume Viewer application on Oscar OOD.\n",
```

tdivoll commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "- Unzip the file and open the Volume viewer application.\n",
+    "\n",
+    "- Click on `load file` and navegate to the root volume viewer data folder. Double click the `osom-loader.txt` file.\n",
```
```suggestion
    "- Click on `load file` and navigate to the root volume viewer data folder. Double click the `osom-loader.txt` file.\n",
```

tdivoll reviewed (approved):
Looking good! I just cleaned up the language in places and suggested capitalization for consistency.

kmilo9999 committed:
fix the name of the volumeviewer module in oscar

tdivoll committed:
Add screenshot of terminal

I added this screenshot because the terminal button in OOD (at the bottom), doesn't seem to work. I had to open it from the menu instead.

tdivoll committed:
clarifications and edits to text in README

tdivoll commented:
I ended up making all the changes after walking through the notebook on Oscar. I pushed them up in a couple commits to add to build up the README.

tdivoll committed:
Add an example for novice users

This will help newbies learn how to turn on the transfer function to play with the example data set.

tdivoll committed:
docs: add example to turn on transfer function and provide contact info

tdivoll committed:
Merge chnages from remote:i
Merge branch 'feature-volume-converter' of https://github.com/ridatadiscoverycenter/riddc-jbook into feature-volume-converter

kmilo9999 commented:
Let's wait for @mirestrepo feedback before merging with master

mirestrepo commented on riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb:
```diff
@@ -0,0 +1,433 @@
+    "\n",
+    "![transfer function clip](https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/transfer_function_example.png)\n",
+    "\n",
+    "- For any support or questions with this product, please contact Center for Computation and Visualization support:   \n",
```
```suggestion
    "- For any support or questions with this product, please reach out to The Center for Computation and Visualization via any of the mediums listed [on our help page](https://ccv.brown.edu/help)  \n",
```

 https://ccv.brown.edu/discuss is a dead link

mirestrepo reviewed (approved):
Looks great. I left a suggestion to update before merging as there is a dead link in there

kmilo9999 committed:
Update help/support link

Co-authored-by: Maria Isabel Restrepo <isabelrestre@gmail.com>

kmilo9999 committed:
removed dead link
